Recruitment/JobSpecs/why_would_you_join_us.md

```diff
@@ -0,0 +1,15 @@
+Do you think there’s something lacking in how many companies approach problems, whether they’re people, process, or technology related?
+
+Do you want to help them figure out how to think and work differently?
+
+Do you love technology, and understand that real change comes from people doing different things, rather than people doing the same things differently?
+
+These are the things that drive us. Do they drive you?
+
+### Who we are looking for
+
+We are looking for people who want to help us change the way companies think and approach problems; people who want to work with teams and individuals who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.

```

Comment:
"poor customer experience" ends the sentence in a negative. The alternative suggested is more positive and also gives another hint into how we like to work. Could be shortened to just "improving customer experience" if we don't want something that long.